src/pages/index.js

```diff
@@ -118,22 +169,73 @@ function closeModal(modal) {
   modal.removeEventListener("mousedown", handleModalClose);
 }
 
+// api handlers
 function handleEditFormSubmit(evt) {
   evt.preventDefault();
-  profileName.textContent = editModalNameInput.value;
-  profileDescription.textContent = editModalDescriptionInput.value;
-  closeModal(editModal);
+  api
+    .editUserInfo({
+      name: editModalNameInput.value,
+      about: editModalDescriptionInput.value,
+    })
+    .then((data) => {
+      profileName.textContent = data.name;
+      profileDescription.textContent = data.about;
+      closeModal(editModal);
+    })
+    .catch(console.error);
 }
```

Comment:
When I submit the **profile** form, the save button should change the text as well

And in `finally` it should be returned back


```js

    .finally(() => {
      submitCardBtn.textContent = "Save";
    });
```